src/main/java/io/github/olgazskiba/Java_blog/random/RandomGeneratorDemo.java

```diff
@@ -0,0 +1,16 @@
+package io.github.olgazskiba.Java_blog.random;
+
+public class RandomGeneratorDemo {
+    public static int random(int randomNumber) {
```

Comment:
I do not understand the logic of randomness if we return the same number as we return